test/SlowTests/Issues/RavenDB_18163.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Queries.Timings;
+using Raven.Server;
+using Raven.Server.Documents;
+using Raven.Server.Documents.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Raven.Tests.Core.Utils.Entities;
+using Raven.Server.Config;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_18163 : RavenTestBase
+    {
+        public RavenDB_18163(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CreatingARevisionManuallyAfterEnablingRevisionsForAnyCollection()
+        {
+            var store = GetDocumentStore();
```

Comment:
`using`

Reply:
https://github.com/ravendb/ravendb/blob/77f7e1201b34aa0ae1d77f78bad1a7fce2e4a906/test/SlowTests/Issues/RavenDB_18163.cs#L79

Reply:
nope, `using` - because if the test fail, you'll leak it.
Always use `using`

Reply:
Done